README.md

```diff
@@ -1 +1,26 @@
 # hw01
+
+## Edited by Xiufan Wu to include a brief biography
+
+Hello, my name is **Xiufan Wu** and I am a second-year MA student in the *Committee on International Relations*. My research interests lies in the state-formation process of East Asian countries, and while I am mostly doing qualitative methods for my MA thesis, I am looking forward to introducing some quantitative elements to it. 
```

Comment:
Nice! That's also what I am thinking. Mainly focused on qualitative methods in my undergraduate years but now I am exploring some quantitative skills. Hope your research goes well!